consensus/istanbul/core/roundstate_db.go

```diff
@@ -283,26 +309,33 @@ func (rsdb *roundStateDBImpl) UpdateLastRoundState(rs RoundState) error {
 	logger := rsdb.logger.New("func", "UpdateLastRoundState")
 	viewKey := view2Key(rs.View())
 
+	before := time.Now()
 	entryBytes, err := rlp.EncodeToBytes(rs)
+	rsdb.rsRLPEncTimer.UpdateSince(before)
```

Comment:
Down in the code the metric is updated after the error check, here it's the opposite. If there's any particular reason for that, let's document it here, if not, standardize on one

Reply:
Good point, updated.